warehouse/packaging/models.py

```diff
@@ -92,6 +93,10 @@ class Project(db.ModelBase):
         nullable=False,
         server_default=sql.func.now(),
     )
+    updated = Column(
+        DateTime(timezone=False),
+        onupdate=datetime.datetime.now
```

Comment:
I'd rather handle this inside the database using a trigger and set this to ``server_onupdate=FetchedValue()`` so that it knows to refetch it.